Resources/doc/installation.rst

```diff
@@ -1,14 +1,16 @@
-Doctrine Bundle
-===============
+Installation
+============
 
 Doctrine DBAL & ORM Bundle for the Symfony Framework.
 
-Because Symfony 2 does not want to force or suggest a specific persistence solutions on the users
-this bundle was removed from the core of the Symfony 2 framework. Doctrine2 will still be a major player
-in the Symfony world and the bundle is maintained by developers in the Doctrine and Symfony communities.
+Because Symfony does not want to force or suggest a specific persistence
+solutions on the users this bundle was removed from the core of the Symfony
+framework. Doctrine2 will still be a major player in the Symfony world and the
+bundle is maintained by developers in the Doctrine and Symfony communities.
 
-IMPORTANT: This bundle is developed for Symfony 2.1 and up. For Symfony 2.0 applications the DoctrineBundle
-is still shipped with the core Symfony repository.
+IMPORTANT: This bundle is developed for Symfony 2.1 and up. For Symfony 2.0
+applications the DoctrineBundle is still shipped with the core Symfony
+repository.
```

Comment:
this warning should be removed:

- Symfony 2.0 does not exist anymore
- the latest versions are not compatible with 2.1 anymore actually. They require 2.3+.
- handling compatibility between versions is best left to Composer anyway